content/docs/user-guide/scripting/lua/debugging-tutorial.md

```diff
@@ -1,171 +1,144 @@
+O3DE **Lua Editor** (Lua IDE) offers an intuitive integrated development environment (IDE) that makes it easy to author, debug, and edit Lua scripts when you create or extend your game. Lua Editor is a standalone application, but can be opened directly from **O3DE Editor** using the Tools menu.
 
 ## Tutorial: Using Lua Editor for Debugging with O3DE Editor 
 
-{{< todo issue="https://github.com/o3de/o3de.org/issues/671" >}}
-Update this Lua debugging tutorial with newer instructions that do not require the Samples Project.
-{{< /todo >}}
+This tutorial shows you how to use O3DE Lua Editor to perform debugging operations on a sample script.
 
-This tutorial shows you how to use O3DE Editor to create a sample level in the **SamplesProject** project with a component entity that contains a Lua script component. You will learn how to open the script in Lua Editor and perform some sample debugging steps on the script.
+1. Open the **Lua Editor** from the **Tools** menu.  
 
-**Debugging Lua scripts using Lua Editor**
+1. Select **New** from the **File** menu to create a new Lua script.
 
-1. Set the **SamplesProject** as the default project using **Project Manager**.
+1. **Copy** and **Paste** the following code into the new script.
 
-1. In O3DE Editor, create a new level by performing *one* of the following steps:
-   + In the **Welcome to O3DE Editor** window, click **New level**
-   + Click **File**, **New**
-   + Press **Ctrl+N**
+    ```lua
+    -- ConstantRotation.lua
+    
+    local ConstantRotation =
+    {
+        Properties =
+            {
+            Rotation = { default = Vector3(0, 0, 90), description = "Constant rotation (in degrees) to apply over time." },
+            },
+    }
 
-1. In the **New Level** dialog box, give the level a name, and then click **OK**.
+    function ConstantRotation:OnActivate()
+        self.rotationRadians = self.Properties.Rotation;
+        self.rotationRadians.x = Math.DegToRad(self.rotationRadians.x);
+        self.rotationRadians.y = Math.DegToRad(self.rotationRadians.y);
+        self.rotationRadians.z = Math.DegToRad(self.rotationRadians.z);
+        self.tickBusHandler = TickBus.Connect(self)
+    end
 
-1. In **Asset Browser**, expand **SamplesProject**, **Objects**, and **SamplesAssets**.
+    function ConstantRotation:OnTick(deltaTime, timePoint)
+        TransformBus.Event.RotateAroundLocalX(self.entityId, self.rotationRadians.x * deltaTime);
+        TransformBus.Event.RotateAroundLocalY(self.entityId, self.rotationRadians.y * deltaTime);
+        TransformBus.Event.RotateAroundLocalZ(self.entityId, self.rotationRadians.z * deltaTime);
+    end
 
-1. Drag **mover\_display\_smooth.cgf** to the perspective viewport.
+    function ConstantRotation:OnDeactivate()
+        self.tickBusHandler:Disconnect()
+    end
+
+    return ConstantRotation
+    ```
+    
+1. **Save** the script as `ConstantRotation.lua` in your project's directory.
+
+1. **Close** the Lua Editor.
```

Comment:
Although these tool names are a bit generic, we are treating them as proper nouns.

```suggestion
1. **Close** Lua Editor.
```